lib/instances/Workspace.lua

```diff
@@ -13,4 +13,6 @@ end
 
 Workspace.properties.CurrentCamera = InstanceProperty.normal({})
 
+Workspace.properties.DistributedGameTime = InstanceProperty.normal({})
```

Comment:
This probably needs a default value of `0` and probably made read-only

Reply:
Just a fun fact, DistributedGameTime isn't actually read only, but it's changed back to the real value immediately afterwards. Roblox issue that Lemur shouldn't emulate.